cmd/zpios/zpios_util.c

```diff
@@ -260,12 +263,13 @@ set_noise(uint64_t *noise, char *optarg, char *arg)
 int
 set_load_params(cmd_args_t *args, char *optarg)
 {
-	char *param, *search, comma[] = ",";
+	char *param, *search, *relief, comma[] = ",";
```

Comment:
I'd suggest calling this something like `searchdup` instead which is more descriptive.  It's a little long for the usage here that's fine.